changelog.d/350.feature

```diff
@@ -0,0 +1 @@
+Add optional UserActivityTracker and BridgeBlocker
```

Comment:
Let's elaborate what these do, so the layman can tell if they want to upgrade or not.

Reply:
How about something like "Add optional UserActivityTracker for tracking & reporting monthly active users, and BridgeBlocker allowing for locking down the bridge communications (intended to be used together)"?

Reply:
Done in 8d9747001019021be8706865a6cd1a8a2c5d52f6